stmt/stmts/insert_test.go

```diff
@@ -96,3 +96,30 @@ func (s *testStmtSuite) TestInsert(c *C) {
 	_, err = s.testDB.Exec(`insert into insert_test (id, c2) values(1, 1) on duplicate key update t.c2 = 10`)
 	c.Assert(err, NotNil)
 }
+
+func (s *testStmtSuite) TestInsertAutoInc(c *C) {
+	createSQL := `drop table if exists insert_autoinc_test; create table insert_autoinc_test (id int auto_increment, c1 int);`
```

Comment:
Seems MySQL need primary key to set auto_increment spec?
This create table statement should be failed.